### DISCUSSION TITLE
Object lists

### DISCUSSION THREAD

**Discussion by maxibenner** [avitorio/outstatic#232] [Ideas]
I would love to use Outstatic for a landing page but the main feature I am missing is to create lists of structured data. Something like this.
`
testimonials: [
   {
      title: "test" ,
      body: "test"
   },
   {
      title: "test" ,
      body: "test"
   }
]
`

Is this something you have on the roadmap or are you focusing on blog-like pages?

**Comment by avitorio** [comment]
It's loosely on the roadmap. What we haven't found is a good UI to create structured data from the interface. 
Have you ever seen something like that?

**Reply by maxibenner** [reply]
I might have expressed this wrong. What I was thinking is basically a repeater for a group of custom fields. No raw editing of structured data.

Since my current project is just a single landing page I resorted to creating a separate collection for repeating objects like the testimonials. But I imagine this would be hard to scale in the context of a multi page site.

**Reply by avitorio** [reply]
This is a great idea, and I have considered it in the past. Creating "sub custom fields". We will definitely support this eventually. I think we would need to rethink how the editor sidebar would look in these cases, probably have a modal to duplicate/edit these fields...

I'd be happy to turn this into a feature + pr.

**Reply by maxibenner** [reply]
Great! In case you are looking for support I could ask my work if they'd be open to me spending some time on this as well.

**Reply by avitorio** [reply]
That'd be awesome. Keep me posted. Feel free to reach out over discord.
